### PR TITLE
chore: 8.0.0 alpha release notes

### PR DESCRIPTION
**Pull request by ecormany**
# Overview

Release notes for 8.0.0.

## Test Plan and Hands on Testing

Check display when alpha0 is cut.

## Changelog

The usual two files.

## Review requests

Any features, bugs, or known issues that need to be added?

## Risk assessment

nil

### CODE REVIEW

File: api/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,30 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
 
 ---
 
+## Opentrons Robot Software Changes in 8.0.0
```
Comment (shlokamin) [inline api/release-notes.md:11]: we need to [add something](https://opentrons.atlassian.net/browse/AUTH-535) about the new EULA right?

File: api/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,30 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
+Welcome to the v8.0.0 release of the Opentrons robot software!
+
+### New Features
+
+- Adds support for storing and running quick transfers.
+- Detect the presence or absence of liquid in a well (Flex pipettes only), and continue or pause the protocol based on the result.
+
+### Improved Features
+
+- Define and use CSV runtime parameters in Python protocols.
```
Comment (shlokamin) [inline api/release-notes.md]: i guess this is an improvement but it's big enough that it feels like a new feature

Comment (sanni-t) [reply api/release-notes.md]: I agree. I see CSV params as a new feature rather than an improvement.

File: api/release-notes.md [hunk 3]
```diff
@@ -4,6 +4,24 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
+
+## Opentrons Robot Software Changes in 8.0.0
+
+Welcome to the v8.0.0 release of the Opentrons robot software!
+
+### New Features
+
+- Create, store, and run quick transfers.
+- Define and use CSV runtime parameters in Python protocols.
+- Detect the presence or absence of liquid in a well (Flex pipettes only), and continue or pause the protocol based on the result.
```
Comment (sfoster1) [inline api/release-notes.md:19]: - Pause protocol execution on overpressure errors and allow recovery and resume

File: api/release-notes.md [hunk 4]
```diff
@@ -4,6 +4,24 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
+
+### Improved Features
+
+- Configure more partial tip pickup configurations. All multi-channel pipettes now support single and partial column pickup, and the Flex 96-channel pipette now supports row pickup.
```
Comment (sfoster1) [inline api/release-notes.md]: - Better handling of situations where liquid is in a tip at the end of a protocol

**Review by sfoster1 — COMMENTED** [review]
the first inline one is that I think overpressure error handling is worth calling out; the second is improvements to drop tip wiz

File: api/release-notes.md (hunk 4 above)
Comment (ecormany) [reply api/release-notes.md]: good call. should we mention that as an app feature too? or is this really that the wizard is the same but the behavior is better?

File: api/release-notes.md [hunk 5]
```diff
@@ -4,6 +4,26 @@ log][]. For a list of currently known issues, please see the [Opentrons issue tr
...
+- Create, store, and run quick transfers on Flex.
+- Define and use CSV runtime parameters in Python protocols.
+- Detect the presence or absence of liquid in a well (Flex pipettes only), and continue or pause the protocol based on the result.
+- Automatically pause Flex protocol runs when detecting overpressure, allowing for error recovery and run resumption.
+
+### Improved Features
+
+- Provides more partial tip pickup configurations. All multi-channel pipettes now support single and partial column pickup, and the Flex 96-channel pipette now supports row pickup.
+- Improves homing behavior when a Flex protocol completes or is canceled with liquid-filled tips attached to the pipette.
+
```
Comment (y3rsh) [inline api/release-notes.md:26]: Ongoing improvement to deck map?

Comment (y3rsh) [reply api/release-notes.md:26]: nvm

File: api/release-notes.md (hunk 4 above)
Comment (sfoster1) [reply api/release-notes.md]: The wizard is technically different than it used to be, but not UI wise. Maybe let's do both just to make it clear. It's a high-priority issue anyway, doesn't hurt to say it twice

**Review by y3rsh — APPROVED** [review]
approved for the alpha.0 cut

**Review by sfoster1 — APPROVED** [review]
Looks good to me for alpha 0!

**Pull request merged**